controller/uninstall_controller.go

```diff
@@ -70,8 +70,8 @@ func NewUninstallController(
 	c := &UninstallController{
 		baseController: newBaseControllerWithQueue("longhorn-uninstall", logger,
 			workqueue.NewNamedRateLimitingQueue(workqueue.NewMaxOfRateLimiter(
-				workqueue.NewItemExponentialFailureRateLimiter(500*time.Millisecond, 5*time.Second),
-				&workqueue.BucketRateLimiter{Limiter: rate.NewLimiter(rate.Limit(10), 100)},
+				workqueue.NewItemExponentialFailureRateLimiter(100*time.Millisecond, 2*time.Second),
```

Comment:
We can just use `EnhancedDefaultControllerRateLimiter` unless it doesn't fit for this scenario.

Reply:
The parameter (max delay is 1000s) of `NewItemExponentialFailureRateLimiter` in `EnhancedDefaultControllerRateLimiter` doesn't fit the uninstaller. You can see the issue post for details.

Reply:
Should we have a max delay of 1000s for other controllers? It was copied from `DefaultControllerRateLimiter`.

Reply:
In other controllers, the keys are typically different. Hence the `EnhancedDefaultControllerRateLimiter ` should be fine.
But in the uninstall controller, the key is always `uninstall`. Keeping enqueuing the same key will be considered as handling failure then trigger the exponential backoff. Then the uninstaller may get blocked.